Mathlib/AlgebraicGeometry/EllipticCurve/VariableChange.lean

```diff
@@ -189,6 +189,38 @@ lemma variableChange_a₆ : (C • W).a₆ =
     C.u⁻¹ ^ 6 * (W.a₆ + C.r * W.a₄ + C.r ^ 2 * W.a₂ + C.r ^ 3 - C.t * W.a₃ - C.t ^ 2
       - C.r * C.t * W.a₁) := rfl
 
+lemma variableChange_inv_a₁ : (C⁻¹ • W).a₁ = C.u * W.a₁ - 2 * C.s := by
+  simp [VariableChange.inv_def]
+  field_simp
+  ring
```

Comment:
There is a terminal `simp` here (and following results). Maybe you can change them before finalizing this PR? You may use the custom simp macro (see the beginning of this file).

Reply:
I think `simp` followed by `field_simp` and `ring` are fine according to mathlib policy. This is because `field_simp` and `ring` are versatile tactics that could absorb the variances in goals that simp-set changes might cause (compare to `rw` or `refine` where the goal needs to be in a very specific state to work).